lib/submission/submitter-payload.js

```diff
@@ -1,4 +1,18 @@
+const {getInstanceProperty} = require('../service-data/service-data')
+
 const generateEmail = (recipientType, from, subject, url, submissionId, addAttachment = true, pdfData = {}) => {
+  const emailBody = recipientType => {
+    let body
+    if (recipientType === 'team') {
+      body = getInstanceProperty('service', 'emailTemplateTeam') || 'Please find an application attached'
```

Comment:
can this be passed into this function by the caller? I think getInstanceProperty might already be used there